task/04-arrays-tasks.js

```diff
@@ -286,7 +315,7 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+   return arr.splice(-3).reverse();
```

Comment:
It will not work if initial array is not sorted. Please, make the solution more universal. Add sort method